contrib/compose/nginx/registry.conf

```diff
@@ -0,0 +1,36 @@
+# For versions of nginx > 1.3.9 that include chunked transfer encoding support
+# Replace with appropriate values where necessary
```

Comment:
The version of nginx used here is known (as it can be specified in the Dockerfile), so this comment isn't strictly required. Unless this configuration doubles as an example for people deploying on bare metal?